crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -297,132 +291,6 @@ async fn two_limit_orders_test(web3: Web3) {
     assert!(balance_after_b.checked_sub(balance_before_b).unwrap() >= to_wei(2));
 }
 
-async fn mixed_limit_and_market_orders_test(web3: Web3) {
```

Comment:
Ditto, this test case still seems useful

Reply:
If we remove market order as we do not support it anymore, we are left with 1 limit order (which is actually in-market) and we already have this test covered  with `local_node_single_limit_order`